apollo-client/src/main/java/com/ctrip/framework/apollo/spring/boot/ApolloApplicationContextInitializer.java

```diff
@@ -140,7 +141,12 @@ protected void initialize(ConfigurableEnvironment environment) {
 
       composite.addPropertySource(configPropertySourceFactory.getConfigPropertySource(namespace, config));
     }
-
+    if (environment.getPropertySources().contains(StandardEnvironment.SYSTEM_ENVIRONMENT_PROPERTY_SOURCE_NAME)) {
+      if (!isOverrideSystemProperties(environment)) {
```

Comment:
how about we add `ApolloClientSystemConsts.APOLLO_OVERRIDE_SYSTEM_PROPERTIES` to the `APOLLO_SYSTEM_PROPERTIES` array in line 85, so that we don't use configUtil.isOverrideSystemProperties() to do the check instead?